### ISSUE TITLE
Potential addition of tracking issue for torch.nn mappings?

### ISSUE THREAD

**Issue by AndreSlavescu** [janestreet/torch#5]
Hi, I'm a recent contributor (PR can be found here: https://github.com/janestreet/torch/pull/4) to this awesome project, and had an idea for having a **torch.nn** layer mapping tracking issue.
Would this be something on the roadmap for this project's development? Would love some feedback for the idea. Below is a skeleton example of how it might look like:

# torch.nn layer mappings 

files: [layer.ml](https://github.com/janestreet/torch/blob/master/src/torch/layer.ml), [layer.mli](https://github.com/janestreet/torch/blob/master/src/torch/layer.mli)

## convolutional layers

- [x] conv1d
- [x] conv2d
- [x] conv3d
- [x] conv1d_transpose
- [x] conv2d_transpose
- [x] conv3d_transpose 

## normalization layers

- [x] batch_norm2d
- [x] instance_norm2d
- [x] layer_norm
- [ ] group_norm 

## non-linear activations

- [x] Relu
- [x] Leaky Relu
- [x] Tanh

etc. 

Open to any feedback on this idea.

**Comment by mwlon** [comment]
@AndreSlavescu You are certainly welcome to maintain this as a tracking issue for missing layer APIs.

I think our approach for this project is to add features on an as-needed/requested basis, though. I wouldn't expect us to actively implement these simply because they are missing.

**Comment by mwlon** [comment]
Closing due to inactivity.
